packages/core/src/odata/v2/odata-batch-request-builder.ts

```diff
@@ -252,7 +251,7 @@ function partitionBatchResponse(response: string, lineBreak: string): string[] {
   }
 
   // E.g., --batch_1234
-  const partSeparator = head(response.split(lineBreak, 1));
+  const partSeparator = response.split(lineBreak, 1)[0];
```

Comment:
I am confused but the limit and [0]. This means the results array contains only on element and this is the one we take, right? I wonder when can the if (!partSeparator) case happen. If the response is defined the split will always return an element.

Reply:
Yeah don't look at this. I removed that in my other PR ;) Here I really just replaced last and head.

Reply:
![Uploading image.png…]()